tests/sanity/test_whereabouts.py

```diff
@@ -41,9 +41,23 @@ def test_whereabouts_rock():
     _run_in_docker(image, True, "ls", "-la", *ROCK_EXPECTED_FILES)
 
     # check binary name and version.
+    process = subprocess.run(
```

Comment:
nit: you could use the `run` from the `k8s-test-harness`

Reply:
Done, now depends on: https://github.com/canonical/k8s-test-harness/pull/11